main/main.go

```diff
@@ -41,11 +41,16 @@ func main() {
 		os.Exit(1)
 	}
 
-	nodeApp := app.New(nodeConfig) // Create node wrapper
 	if term.IsTerminal(int(os.Stdout.Fd())) {
 		fmt.Println(app.Header)
```

Comment:
Unrelated to this PR but possibly a followup - feels like this should be outputted into the logger instead of using `fmt.Println`.